pages/common/git-column.md

```diff
@@ -0,0 +1,16 @@
+# git column
+
+> Display data in columns.
+> More information: <https://git-scm.com/docs/git-column>.
+
+- List files and folders in the current directory, formatted as columns:
```

Comment:
Generally, we try to avoid demonstrating the functionality of other commands in a tldr page. Can we reword this to be more focused on the functionality of the current subcommand at all?
```suggestion
- Format the standard input as multiple columns:
```

Same goes for the other examples.